operator/internal/manifests/internal/gateway/build_test.go

```diff
@@ -195,6 +195,15 @@ tenants:
   opa:
     url: http://127.0.0.1:8080/v1/data/lokistack/allow
     withAccessToken: true
+- name: network
+  id: 3e922593-e352-47df-8c5c-c39dbdd5b83c
+  openshift:
+    serviceAccount: lokistack-gateway
+    redirectURL: https://localhost:8443/openshift/network/callback
+    cookieSecret: whynot
+  opa:
+    url: http://127.0.0.1:8080/v1/data/lokistack/allow
+    withAccessToken: true
```

Comment:
I strongly believe this should be a separate test case where you separately configure the `Options.Stack.Tenants.Mode: lokiv1.OpenShiftNetwork` and you get a config rendering with only a single tenant namely `network`.